app/electron.js

```diff
@@ -21,8 +21,8 @@ const staticWindows = {
   },
   help : {
     config : {
-      height          : 625,
-      width           : 625,
+      height          : 400,
+      width           : 550,
```

Comment:
@radibit This was a missunderstanding. Was too lazy in sketch to adjust that. :)